blog/introducing-api-history.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Introducing API History (GSoC 2024)
+date: 2024-08-20T00:00:00.000Z
+authors: piotrpdev # Maybe don't use global author?
+slug: introducing-api-history
+tags: [community]
+---
+
+New breaking changes in Electron APIs will now be displayed under those APIs in the docs.
+
+---
+
+## What is API History?
+
+Hi 👋, I'm Peter, the 2024 [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/)
+contributor to Electron.
+
+Over the course of the GSoC program I implemented an API history feature for the
+Electron documentation and its functions, classes, etc. in a similar fashion to the
+[Node.js documentation](https://nodejs.org/en/docs): by allowing the
+use of a simple but powerful YAML schema in the API documentation Markdown files
+and displaying it nicely on the Electron documentation website.
+
+### Details
```

Comment:
```suggestion
```

I think it'd be a slight improvement to flatten the heading structure in this part.